yaksh/documentation/moderator_docs/creating_course.rst

```diff
@@ -18,18 +18,24 @@ Setting up a new course
 Features in Courses
 -------------------
 
-	 Click on the Courses link located at the navigation bar, to view all the courses created by the moderator.
+	 Click on the Courses link located at the navigation bar.
 
 	 .. image:: ../images/course_features.jpg
 
+	 This page shows all the courses created by a moderator and all the courses allotted to a moderator.
```

Comment:
Could use a picture of courses' page showing courses for both alloted and created courses